schema/operation.json

```diff
@@ -3,37 +3,118 @@
     "$id": "http://standards.ncats.io/operation.json",
     "anyOf": [
         {
-            "$ref": "#/$defs/OperationOverlay"
+            "$ref": "#/$defs/OperationCompleteResults"
```

Comment:
These are sorted in #27. After that is merged, we can rebase this to fix the diff.